update reuse license and docs (#400)
PyPSA/atlite: doc/installation.rst

..
  SPDX-FileCopyrightText: Contributors to Atlite <https://github.com/pypsa/atlite>

  SPDX-License-Identifier: CC-BY-4.0

##############
Installation
##############

Getting Python
==============

If it is your first time with Python, we recommend `conda

> <https://docs.conda.io/en/latest/miniconda.html>`_, `mamba
> <https://github.com/mamba-org/mamba>`_ or `pip

Reviewer: ```suggestion
<https://docs.conda.io/en/latest/miniconda.html>`_ .
```

Let's get rid of `mamba`. The solver has been incorporated as default into `conda` already and we don't want to confuse the first time Python project users.

> <https://docs.conda.io/en/latest/miniconda.html>`_, `mamba
> <https://github.com/mamba-org/mamba>`_ or `pip

Reviewer: ```suggestion
<https://docs.conda.io/en/latest/miniconda.html>`_ or `pip
```

Let's get rid of `mamba`. The solver has been incorporated as default into `conda` already and we don't want to confuse the first time Python project users.

> <https://pip.pypa.io/en/stable/>`_ as easy-to-use package managers. They are

Reviewer: ```suggestion
`pip <https://pip.pypa.io/en/stable/>`_ can also be used, but may cause problems with some GIS related Python packages.
```

Author: Is this the case? Tests right now run on `pip`/ `uv`

Reviewer: See here: https://conda.org/blog/2023-11-06-conda-23-10-0-release/

Author: No, I mean regarding `pip` which "may cause problems with some GIS related Python packages."

> available for Windows, Mac OS X and GNU/Linux.

Reviewer: ```suggestion
Both are easy-to-use package managers and available for Windows, Mac OS X and GNU/Linux.
```

It is always helpful to use dedicated `conda/mamba environments <https://mamba.readthedocs.io/en/latest/user_guide/mamba.html>`_ or `virtual environments
<https://pypi.python.org/pypi/virtualenv>`_.

Installation with conda
=======================

If you are using ``conda`` you can install PyPSA with::

    conda install -c conda-forge atlite

> Replace ``conda`` with ``mamba`` if you use this alternative.

Reviewer: ```suggestion
```
